guia/closures.md

```diff
@@ -1 +1,131 @@
-# Closures
+## Closures
+
+*Closures* são blocos de funcionalidade independentes que podem ser passados e usados em seu código. *Closures*, em Swift, são similares aos blocos em C e Objective-C e a *lambdas* em outras linguagens de programação.
+
+*Closures* podem capturar e armazenar referências para quaisquer constantes e variáveis do contexto no qual ele foi definido. Isso é conhecido como *closure* sobre constantes e variáveis. O Swift se encarrega de realizar o gerenciamento de memória de captura por você.
```

Comment:
No original, temos "This is known as _closing over_ those constants and variables.". Acho que a tradução "...como *closure* sobre constantes..." ficou estranha, IMHO. _Closure_ aqui foi usado como verbo, não como o conceito de computação. Num exemplo bem grotesco, seria como falar: "Por favor, _print_ aquele documento para mim." - e esse não foi um exemplo bom, porque está traduzindo parte de algo que é uma expressão: "_closing over_".

Eu traduziria como "Isso é conhecido como _fechar sobre_ constantes e variáveis", mesmo a votação no Slack indicar que o pessoal prefere "closure_ do que fechamento.